bin/tests/it/cli.rs

```diff
@@ -961,7 +962,7 @@ fn lookback_tail_lines_file_created_after_agent_start_at_beg() {
         let (line_count, server) = tokio::join!(
             async {
                 let mut handle = common::spawn_agent(AgentSettings {
-                    log_dirs: &dir_path,
+                    log_dirs: &log_path,
```

Comment:
Changed this so the tempdir does not get moved into this block. Previously this was causing the tempdir to be dropped immediately after we were done checking the agents log output. The agent would see it's logging directory delete, force a rescan, and panic because the logging directory isn't a directory.

This all happens before the lines get sent to the mock ingester which are checked afterwards.

Not sure why my changes is causing this problem to present itself now - maybe fs operations are faster now.